apps/aecore/test/aec_peers_tests.erl

```diff
@@ -282,21 +282,21 @@ test_multiple_normal_peers() ->
     Id3 = aec_peers:peer_id(PubKey3),
 
     Peers = [
-        peer(PubKey1, <<"aeternity.com">>, 4000),
-        peer(PubKey2, "google.com", 4000),
-        peer(PubKey3, "192.168.0.1", 4000)
+        peer(PubKey1, <<"10.1.0.1">>, 4000),
+        peer(PubKey2, "10.2.0.1", 4000),
+        peer(PubKey3, "10.3.0.1", 4000)
     ],
 
     aec_peers:add_peers(Source, Peers),
     %% First one is connected to right away, the second after 1 second
     %% and the third one after 2 seconds; but there is no way of knowing the
     %% order they will get connect to.
     A = erlang:system_time(millisecond),
-    {ok, Conn1} = ?assertCalled(connect, [#{ r_pubkey := PubKey1 }], {ok, _}, 3150),
-    {ok, Conn2} = ?assertCalled(connect, [#{ r_pubkey := PubKey2 }], {ok, _}, 3150),
-    {ok, Conn3} = ?assertCalled(connect, [#{ r_pubkey := PubKey3 }], {ok, _}, 3150),
+    {ok, Conn1} = ?assertCalled(connect, [#{ r_pubkey := PubKey1 }], {ok, _}, 3500),
+    {ok, Conn2} = ?assertCalled(connect, [#{ r_pubkey := PubKey2 }], {ok, _}, 3500),
+    {ok, Conn3} = ?assertCalled(connect, [#{ r_pubkey := PubKey3 }], {ok, _}, 3500),
     B = erlang:system_time(millisecond),
-    ?assert((B - A) =< 3150),
+    ?assertEqual({(B - A), true}, {(B - A), ((B - A) =< 3500)}),
```

Comment:
I understand why the assert does what it does (showing the actual value when failing) but perhaps the `B - A` expression could be assigned to a variable to make it a bit cleaner?